pkgs/applications/graphics/veusz/default.nix

```diff
@@ -0,0 +1,71 @@
+{ python3Packages
+, qtbase
+, ghostscript
+, wrapQtAppsHook
+, lib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "veusz";
+  version = "3.3.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "4ClgYwiU21wHDve2q9cItSAVb9hbR2F+fJc8znGI8OA=";
+  };
+
+  nativeBuildInputs = [ wrapQtAppsHook python3Packages.sip ];
+
+  buildInputs = [ qtbase ];
+
+  dontWrapQtApps = true;
+  preFixup = ''
+    wrapQtApp "$out/bin/veusz"
+  '';
```

Comment:
Why does the auto wrapping not work for us?

Reply:
If I comment out these lines, I get the error
```
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```
I'm fairly certain this is because `wrapQtAppsHook` ignores non-ELF executables (and veusz is a `/bin/bash -e` script), as described in the manual [here](https://nixos.org/manual/nixpkgs/stable/#qt-runtime-dependencies).

Reply:
I've added a comment in the code to describe why I did this :)